packages/meta-harness/lib/meta_harness_common.py

```
@@ -0,0 +1,943 @@
+
+
+class LockAcquisitionError(RuntimeError):
+    """store.lock が取得できない場合に送出する（CLI は exit 3）。"""
+
+
+_LOCK_ACQUIRE_ATTEMPTS = 2
+
+
+@contextmanager
+def store_lock(main_root: Path, config: dict):
+    """store.lock を取得するコンテキストマネージャ（短期 TTL、Sec2-3）。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)